src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/InitialConsiderationService.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class InitialConsiderationService {
+    private static final String HORIZONTAL_RULE = "<hr>";
+
+    private static final String completeICTitle = "<h3>What happens next</h3>";
+    private static final String completeICText =

```

Comment:
Constants should be named with upper case - e.g. COMPLETE_IC_TITLE

Reply:
fixed